### PR TITLE
289 - Optimise CSS bundle changes

### LINKED ISSUE #289: [CSS] Cleanup css and avoid using @apply
Blocks we can get the CSS files removed or at least reduced:

* https://github.com/hlxsites/danaher-ls-aem/tree/main/blocks/breadcrumb
* https://github.com/hlxsites/danaher-ls-aem/blob/main/blocks/cards
* https://github.com/hlxsites/danaher-ls-aem/blob/main/blocks/columns
* https://github.com/hlxsites/danaher-ls-aem/tree/main/blocks/hero
* https://github.com/hlxsites/danaher-ls-aem/tree/main/blocks/popular-articles
* https://github.com/hlxsites/danaher-ls-aem/tree/main/blocks/recent-articles

### PR DESCRIPTION
**Pull request by amlan18041996**
Fix #289 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://289-cleanup-css-avoid-apply--danaher-ls-aem--hlxsites.hlx.live

### CODE REVIEW

**Comment by davenichols-DHLS** [comment]
@mhaack - the code change looks good, but from this the CLS is actually slightly worse when I look at Lighthouse?  Not sure if that is just random or something we need to worry about?

File: blocks/breadcrumb/breadcrumb.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default function decorate(block) {
   homeIconLi.innerHTML = '<a href="/" title="Home"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" class="flex-shrink-0 h-5 w-5" data-di-rand="1697430026535"><path d="M11.47 3.84a.75.75 0 011.06 0l8.69 8.69a.75.75 0 101.06-1.06l-8.689-8.69a2.25 2.25 0 00-3.182 0l-8.69 8.69a.75.75 0 001.061 1.06l8.69-8.69z"></path><path d="M12 5.432l8.159 8.159c.03.03.06.058.091.086v6.198c0 1.035-.84 1.875-1.875 1.875H15a.75.75 0 01-.75-.75v-4.5a.75.75 0 00-.75-.75h-3a.75.75 0 00-.75.75V21a.75.75 0 01-.75.75H5.625a1.875 1.875 0 01-1.875-1.875v-6.198a2.29 2.29 0 00.091-.086L12 5.43z"></path></svg></a>';
   listsClone.prepend(homeIconLi);
   Array.from(listsClone.children).forEach((element, index) => {
-    element.className = 'flex items-center gap-x-3 text-sm font-medium text-gray-500 whitespace-nowrap hover:text-gray-700';
+    element.className = ' flex items-center gap-x-3 text-sm font-medium text-gray-500 whitespace-nowrap hover:text-gray-700';
```
Comment (mhaack) [inline blocks/breadcrumb/breadcrumb.js:11]: Space at the begin can be removed.

File: blocks/columns/columns.js [hunk 2]
```diff
@@ -1,4 +1,10 @@
 export default function decorate(block) {
+  if (block.parentElement.parentElement.className.includes('columns-container')) {
+    block.parentElement.parentElement.className += ' px-0 lg:px-8 !py-4 md:!py-10';
```
Comment (mhaack) [inline blocks/columns/columns.js:3]: Instead of `.className +=` we could use `.classList.add('..', '..')`

File: blocks/header/header.js [hunk 3]
```diff
@@ -53,7 +53,7 @@ function setRecentSearches(searchValue) {
 function toggleSearchBoxMobile(e) {
   e.preventDefault();
   const searchBox = document.querySelector('.mobile-search');
-  searchBox.classList.toggle('show');
+  searchBox.classList.toggle('show', '!block');
```
Comment (mhaack) [inline blocks/header/header.js:56]: Is that correct a class starting with `!`?

File: blocks/hero/hero.js [hunk 4]
```diff
@@ -66,6 +66,8 @@ const buildVideoModal = async (href) => {
 };
 
 export default function decorate(block) {
+  block.parentElement.parentElement.className += ' !p-0';
+  block.parentElement.className += ' !max-w-[unset]';
```
Comment (mhaack) [inline blocks/hero/hero.js:70]: Again this could be `.classList.add(..)`

**Comment by rgravitvl** [comment]
@amlan18041996 Please add more blogs/news pages for testing

**Review by davenichols-DHLS — APPROVED** [review]
@mhaack  and @amlan18041996  - this is looking good to me.  I tested pages and other items and they all appear to work as expected:

https://289-cleanup-css-avoid-apply--danaher-ls-aem--hlxsites.hlx.live/
https://289-cleanup-css-avoid-apply--danaher-ls-aem--hlxsites.hlx.live/us/en/blog
https://289-cleanup-css-avoid-apply--danaher-ls-aem--hlxsites.hlx.live/us/en/news 
https://289-cleanup-css-avoid-apply--danaher-ls-aem--hlxsites.hlx.live/us/en/blog/revolutionizing-drug-discovery-with-organoids 
https://289-cleanup-css-avoid-apply--danaher-ls-aem--hlxsites.hlx.live/us/en/blog/how-organoids-can-redefine-pre-clinical-research
https://289-cleanup-css-avoid-apply--danaher-ls-aem--hlxsites.hlx.live/us/en/news/scientists-can-study-neuronal-morphology-harnessing-the-power-of-ai 
https://289-cleanup-css-avoid-apply--danaher-ls-aem--hlxsites.hlx.live/us/en/products/centrifuges
https://289-cleanup-css-avoid-apply--danaher-ls-aem--hlxsites.hlx.live/us/en/products/centrifuges/benchtop-centrifuges

**Review by mhaack — APPROVED** [review]

**Review by duynguyen — APPROVED** [review]

**Pull request merged**